### ISSUE TITLE
The Events Calendar plugin by Modern Tribe - No Page body class

### ISSUE THREAD

**Issue by pingram3541** [pojome/elementor#2029]
## Description

When using custom css of Elementor Pro for an Event post type (and it is an enabled type in Elementor > Settings) it does not apply on the front end if using the dynamic "selector" key word in the css, i.e. `selector .some-class { ... }`.

## Steps to reproduce

1. Activate The Events Calendar plugin
2. Enable "events" post type in Elementor > Settings
3. Create a new event and edit w/ Elementor
4. Edit page custom css save and view front end
5. Notice page css is present but doesn't apply because custom body class doesn't exist - `body.elementor-page-XXX` (i.e. body.elementor-page-postid)

## Environment

Tested multiple hosting products

**Comment by arielk** [comment]
The Custom CSS option works only if the page set as Elementor type, so if you are not using Elementor editor in the page, the Custom CSS doesn't change anything.

You can change the CSS with your theme customizer.

Thanks

**Comment by pingram3541** [comment]
@arielk Did you read what I typed?

> The Custom CSS option works only if the page set as Elementor type

This is why I detailed on line 2 exactly that.  ???

> so if you are not using Elementor editor in the page

Line 3 should tell you that I was indeed using the Elementor editor on the page

**Comment by pingram3541** [comment]
Thank you for re-opening!

**Comment by bainternet** [comment]
@pingram3541 

Doing some house cleaning (trying at least) and I tested the issue described and you are correct the class `.elementor-page-XXX` is missing.
Digging in I found that its actually there but the XXX (post-id) is wrong and somehow being overwritten by Tribe and by the time Elementor adds the body class the post id is not correct.

Sadly there is nothing we can do about it, this should be directed at them.

thanks.
